lua/rustaceanvim/config/init.lua

```diff
@@ -58,17 +58,11 @@ vim.g.rustaceanvim = vim.g.rustaceanvim
 ---@field executor? RustaceanExecutor | executor_alias
 ---@field on_initialized? fun(health:RustAnalyzerInitializedStatus) Function that is invoked when the LSP server has finished initializing
 ---@field reload_workspace_from_cargo_toml? boolean Automatically call `RustReloadWorkspace` when writing to a Cargo.toml file
----@field hover_actions? RustaceanHoverActionsOpts Options for hover actions
```

Comment:
issue: this is not backward compatible.

We should keep the `hover_actions` field and the `RustaceanHoverActionsOpts` class, but the class should have all fields removed, except for `replace_builtin_hover`.

(This needs to be reflected in the changelog)